client/src/page/registration/registration.tsx

```diff
@@ -70,6 +77,18 @@ const Registration = () => {
           required
           error={nameError}
           helperText={nameError ? "Name is required" : ""}
+          InputProps={{
+          style: {
```

Comment:
You shouldn't change theme styles here, go to the FormField component, and apply these changes there

Reply:
ready